cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -54,50 +54,22 @@ string(TOUPPER ${CMAKE_BUILD_TYPE} UPPERCASE_BUILD_TYPE)
 
 if (DEFINED ENV{SNAPPY_HOME})
   set (SNAPPY_HOME "$ENV{SNAPPY_HOME}")
-elseif (Snappy_ROOT)
```

Comment:
@kou I have deleted lines like these so setting `XXX_ROOT` will not set `XXX_HOME` any more and it will skip `FindXXXAlt.cmake` by directly jumping to the `FetchContent` branch.